Update serving tutorial (#4815)
knative/docs: docs/getting-started/first-traffic-split.md

## Splitting Traffic

> You may at this point be wondering, "where did 'Hello World!' go?" Remember, Revisions are a immutable snapshot-in-time of application code and configuration, so your old `hello-00001` Revision is still available to you.

Reviewer: ```suggestion
You may at this point be wondering, "where did 'Hello World!' go?" Remember, Revisions are an immutable snapshot-in-time of application code and configuration, so your old `hello-00001` Revision is still available to you.
```

          traffic:
          - latestRevision: true
            percent: 50
          - latestRevision: false
            percent: 50
            revisionName: hello-00001
        ```
    1. Apply the YAML by running the command:
        ``` bash
        kubectl apply -f hello.yaml
        ```

!!! info
    `@latest` will always point to our "latest" Revision which, at the moment, is `hello-00002`.

### Verify the traffic split

> Verify that the traffic split has configured correctly by listing the revisions again, run the command:

Reviewer: ```suggestion
To verify that the traffic split has configured correctly, list the revisions again by running the command:
```

```bash
kn revisions list
```
!!! Success "Expected output"
    ```{ .bash .no-copy }
    NAME            SERVICE   TRAFFIC   TAGS   GENERATION   AGE   CONDITIONS   READY   REASON
    hello-00002     hello     50%              2            10m   3 OK / 4     True
    hello-00001     hello     50%              1            36m   3 OK / 4     True
    ```

> Access your Knative Service multiple times on the browser to see the different output being served by each Revision.

Reviewer: ```suggestion
Access your Knative Service multiple times in your browser to see the different output being served by each Revision.
```

You can tag a revision and this will create a new url exclusively for the revision.

Lets tag the old revision `hello-00001` as `v1` using the `kn` CLI
```bash
kn service update hello --tag hello-00001=v1
```

You can see the revision now was a tag under the column `TAGS`
```bash
kn revision list hello-00001
```

!!! Success "Expected output"
    ```{ .bash .no-copy }
    NAME          SERVICE   TRAFFIC   TAGS   GENERATION   AGE   CONDITIONS   READY   REASON
    hello-00001   hello     50%       v1     1            15m   3 OK / 4     True
    ```

You can append `v1-` to the Knative Service domain (eg. `http://v1-hello.default.${LOADBALANCER_IP}.sslip.io`) to access this revision, you can get the value of url for the tagged revision from the `status` field on the Kubernetes resource `ksvc`. You can use kubectl to print the yaml representation to read `status`. Use the following `curl` to access the revision
```bash
echo "Accessing URL $(kubectl get ksvc hello  -o yaml | grep '//v1-' | awk -F 'url: ' '{print $2}')"
curl "$(kubectl get ksvc hello  -o yaml | grep '//v1-' | awk -F 'url: ' '{print $2}')"
```

!!! Success "Expected output"
    ```{ .bash .no-copy }
    Hello World!

>     ```

Author: I think showing `Accessing URL http://v1-hello.default.${LOADBALANCER_IP}.sslip.io` this on the doc is confusing, but they will see the real value when they run it, this is just in case they have problems they can see the actual URL being called.

>     ```

Reviewer: My thinking was that when I pasted the commands into the terminal I didn't see `Hello World!` immediately I just saw `Accessing URL...`. I then remembered I had copied 2 commands and had still had to run the second one. 

This might be because I've run the tutorial before so I was used to hello world outputting immediately.

>     ```

Author: I want to avoid the noise in the doc with so much repetitive lines with `${LOADBALANCER_IP}`
